src/redux/viewChange.js

```diff
@@ -95,6 +94,8 @@ export const viewChangeAndFetch = (newView) => (dispatch, getState) => {
     dispatch(stopTrackingLocation())
   }
 
+  // TODO: retrive event from viewChange and change order accordingly?
+
```

Comment:
Is this todo still applicable?

Reply:
This was when I was drafting stuff for the optimization feature... whoops. Will remove this.